src/constants/strings.js

```diff
@@ -262,8 +262,10 @@ export const MANAGE_CLIENT_BANNER_CONTENT =
   'Search for agreement holders and link them to their corresponding client.'
 
 // tips
-export const CONDITIONS_TIP =
+export const CONDITIONS_TIP_DECISION_MAKER =
   'Review and update as needed the proposed conditions that would be imposed under FRPA 112 as part of this decision'
+export const CONDITIONS_TIP_RANGE_OFFICER =
```

Comment:
We can keep it `CONDITIONS_TIP`, and call the new one `PROPOSED_CONDITIONS_TIP`.